tests/unittest/disaggregated/test_bounce.py

```
@@ -0,0 +1,337 @@
+
+import numpy as np
+import pytest
+
+from tensorrt_llm._torch.disaggregation.native.bounce import config as bcfg
+
+# transport/transfer pull CUDA-binding + torch deps at import; skip gracefully if absent.
+try:
+    from tensorrt_llm._torch.disaggregation.native.bounce import transport as btr
+
+    _HAVE_TRANSPORT = True
+except Exception:  # pragma: no cover - import guard for CPU-only envs without CUDA bindings
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)